prac_09/sort_files_2.py

```diff
@@ -0,0 +1,47 @@
+""" Processes the files in the directory and sorts them into their appropriate folders. """
```

Comment:
I would remove the spaces at the start and end of the docstrings.
```suggestion
"""Processes the files in the directory and sorts them into their appropriate folders."""
```